### ISSUE TITLE
Cannot suspend POP OS anymore

### ISSUE THREAD

**Issue by BarbzYHOOL** [pop-os/pop#3087]
**Distribution (run `cat /etc/os-release`):**
 NAME="Pop!_OS"
VERSION="22.04 LTS"

 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

**Issue/Bug Description:**

yesterday i put my computer to sleep/suspend (ubuntu 22.04), when i went to wake it up, it printed a black screen with messages and i couldn't do shit so I did REISU- and then it prompted me a new session

here are some errors i got on that black screen and then afterwards in the logs:
debugfs: File 'radeon_ring_gfx' in directory '0' already present!
debugfs: File 'radeon_ring_cp1' in directory '0' already present!
_cp2
_dma1
_dma2
_uvd
_vce1
_vce2

Here another in the logs:
[drm:si_dpm_set_power_state [radeon]] *ERROR* si_restrict_performance_levels_before_switch failed

**Steps to reproduce (if you know):**

suspend computer and then wake it up, it wont work

**Expected behavior:**

working

**Other Notes:**

this didn't happen for months, and suddenly happened after a big update, maybe kernel related

**Comment by leviport** [comment]
Please describe your hardware setup. Also, does this suspend failure happen in a Pop live disk, or while booted into an up-to-date recovery partition?

**Comment by BarbzYHOOL** [comment]
what do you mean my hardware setup? my motherboard model?

And so you suggest me to update the recovery partition and try this in recovery mode?

**Comment by leviport** [comment]
Yes, motherboard, CPU, GPU, and anything else that might be relevant. And if you have a desktop, it might be worth checking your BIOS settings. If you see anything about S0ix or "modern standby", that might be worth experimenting with.

Checking from a live disk or the recovery partition helps us narrow down whether the issue occurs only in your installation or not. If you're running custom kernels, there could be a lot of variables at play.

**Comment by BarbzYHOOL** [comment]
not running a custom kernel

```


Kernel | Linux 6.2.6-76060206-generic (x86_64)
-- | --
Version | #202303130630~1689015125~22.04~ab2190e SMP PREEMPT_DYNAMIC Mon J
C Library | GNU C Library / (Ubuntu GLIBC 2.35-0ubuntu3.1) 2.35
Distribution | Pop!_OS 22.04 LTS




Processor | AMD FX(tm)-6300 Six-Core Processor
-- | --
Memory | 16332MB (8525MB used)
Machine Type | Desktop



BIOS
--
Date | 04/08/2013
Vendor | American Megatrends Inc. (American Megatrends, www.ami.com)
Version | F1
Board
Name | 970A-DS3P
Vendor | Gigabyte Technology Co., Ltd. (Gigabyte Technology, www.gigabyte.com.tw)

```

**Comment by BarbzYHOOL** [comment]
mb related https://github.com/pop-os/pop/issues/3084

**Comment by BarbzYHOOL** [comment]
@leviport I updated the recovery partition, booted on it, then "suspend" it, and it printed the same error :'(

**Comment by leviport** [comment]
Cool, that means we've ruled out anything specific to your individual install. What GPU are you using?

**Comment by BarbzYHOOL** [comment]
> Cool, that means we've ruled out anything specific to your individual install. What GPU are you using?


Host bridge		: Advanced Micro Devices, Inc. [AMD] Family 15h Processor Function 5
VGA compatible controller		: Advanced Micro Devices, Inc. [AMD/ATI] Pitcairn XT [Radeon HD 7870 GHz Edition] (prog-if 00 [VGA controller])

**Comment by BarbzYHOOL** [comment]
help
